### PR TITLE
Use cff instead of bib

### PR DESCRIPTION
**Pull request by tmigot**
@software{Orban_CUTEst_jl_Julia_s_CUTEst_2023,
author = {Orban, Dominique and Soares Siqueira, Abel and contributors},
month = mar,
title = {{CUTEst.jl: Julia's CUTEst interface}},
url = {https://github.com/JuliaSmoothOptimizers/CUTEst.jl},
version = {0.13.2},
year = {2023}
}

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  - given-names: Dominique
+    family-names: Orban
+    email: dominique.orban@gerad.ca
+    affiliation: >-
+      GERAD and Department of Mathematics and            
+      Industrial Engineering, Polytechnique
+      Montréal,             QC, Canada
+    orcid: 'https://orcid.org/0000-0002-8017-7687'
+  - given-names: Abel Soares
+    family-names: Siqueira
```
Comment (abelsiqueira) [inline CITATION.cff:20]:
```suggestion
  - given-names: Abel
    family-names: Soares Siqueira
```

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  - given-names: contributors
+identifiers:
+  - type: doi
+    value: 10.5281/zenodo.1188851
+repository-code: 'https://github.com/JuliaSmoothOptimizers/CUTEst.jl'
+keywords:
+  - CUTEst
+  - Optimization
+  - Julia
+commit: cf290bc63dd5b2a456ad84a3f7bf8dd089fc4074
```
Comment (abelsiqueira) [inline CITATION.cff:33]:
```suggestion
```

**Review by abelsiqueira — CHANGES_REQUESTED** [review]
Same comments as https://github.com/JuliaSmoothOptimizers/AmplNLReader.jl/pull/131#event-11542388672

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  metadata from this file.
+type: software
+authors:
+  - given-names: Dominique
+    family-names: Orban
+    email: dominique.orban@gerad.ca
+    affiliation: >-
+      GERAD and Department of Mathematics and            
+      Industrial Engineering, Polytechnique
+      Montréal,             QC, Canada
```
Comment (tmigot) [inline CITATION.cff:17]:
```suggestion
    affiliation: >-
      GERAD and Department of Mathematics and            
      Industrial Engineering, Polytechnique
      Montréal, QC, Canada
```

**Pull request merged**